cmd/boost/deals.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	bcli "github.com/filecoin-project/boost/cli"
+	"github.com/google/uuid"
+	"github.com/urfave/cli/v2"
+)
+
+var dealCmd = &cli.Command{
+	Name:  "deal",
+	Usage: "Manage Boost deals",
+	Subcommands: []*cli.Command{
+		makeOfflineDealWithData,
+	},
+}
+
+var makeOfflineDealWithData = &cli.Command{
+	Name:  "offline-deal-with-data",
+	Usage: "Make offline deal with data",
+	Flags: []cli.Flag{&cli.StringFlag{
+		Name:  "dealUuid",
```

Comment:
I think conventionally we use snake-case for param names, ie "deal-uuid"

Reply:
Done.